### PR TITLE
Update get-payflow-link-token.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Added Error Message for the invalid cart id in the query.

## Purpose of this pull request

This pull request (PR) add the Error message for the query if you provides invalid cartId in the input query parameter. This will helpful to user to know the error when they face this type of error while running query.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/graphql/queries/get-payflow-link-token.html
https://devdocs.magento.com/guides/v2.4/graphql/queries/get-payflow-link-token.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/get-payflow-link-token.md [hunk 1]
```diff
@@ -65,3 +65,9 @@ Attribute |  Data Type | Description
 `paypal_url` | String | The PayPal URL used for requesting a Payflow form
 `secure_token` | String | Secure token generated by PayPal
 `secure_token_id` | String | Secure token ID generated by PayPal
+
+## Errors
+
+Error | Description
+--- | ---
+`No such entity with cartId` | If you provide invalid cartId.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/get-payflow-link-token.md:73]:
```suggestion
`No such entity with cartId` | If invalid `cartId` is provided
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a small correction

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have updated the correction. Thank you.

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by keharper — APPROVED** [review]

**Pull request merged**